src/content/1.7/webservice/resources/customizations.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Customizations
+---
+
+# Resources for Customizations
+
+### Customization
+
+|           Name           |    Format    | Required | Read Only | Max size | Not filterable | Description |
```

Comment:
For this kind of API the `Max size` column might be useless as well

Reply:
Maybe, but it's a generated file.
I will try to improve this depending on the values we have in the results.

Reply:
I remove all unneeded columns when it's possible.